keras/backend/tensorflow_backend.py

```diff
@@ -2525,6 +2557,7 @@ def one_hot(indices, num_classes):
     # Returns
         (n + 1)D one hot representation of the input
         with shape `(batch_size, dim1, dim2, ... dim(n-1), num_classes)`
+    {{np_implementation}}
```

Comment:
If I remember correctly, we use `to_categorical` for this one. Since the imports are not added in the docs, users will wonder what `to_categorical` is. Let's remove {{np_implementation}} it until we find another way to display it nicely.

Reply:
I can clearly understand. Thanks to advice.